include/mbedtls/ecp.h

```diff
@@ -1339,8 +1339,6 @@ int mbedtls_ecp_read_key(mbedtls_ecp_group_id grp_id, mbedtls_ecp_keypair *key,
  * \return          \c 0 on success.
  * \return          #MBEDTLS_ERR_ECP_BUFFER_TOO_SMALL if the \p key
                     representation is larger than the available space in \p buf.
- * \return          #MBEDTLS_ERR_ECP_FEATURE_UNAVAILABLE if the operation for
- *                  the group is not implemented.
```

Comment:
Note: I think that comes from the time when we had only the arithmetic (`ecp_mul()`) for Curve25519 but the import/export format had not been standardized yet. So, it made sense at that time, but can indeed be removed now.